iis-workflows/iis-workflows-documentssimilarity/src/test/resources/eu/dnetlib/iis/workflows/documentssimilarity/main_workflow/oozie_app/workflow.xml

```diff
@@ -64,11 +72,9 @@
 			<!-- This is simple wrapper for the Java code -->
 			<main-class>eu.dnetlib.iis.common.java.ProcessWrapper</main-class>
 			<!-- The business Java code that gets to be executed -->
-			<arg>eu.dnetlib.iis.common.java.jsonworkflownodes.TestingConsumer</arg>
+			<arg>eu.dnetlib.iis.common.java.jsonworkflownodes.RecordCountTestConsumer</arg>
 			<!-- Specification of the input ports -->
-			<arg>-C{docsim,
-				eu.dnetlib.iis.documentssimilarity.schemas.DocumentSimilarity,
-				eu/dnetlib/iis/workflows/documentssimilarity/main_workflow/data/documents_similarity.json}</arg>
+			<arg>-C{docsim,eu.dnetlib.iis.documentssimilarity.schemas.DocumentSimilarity,1,750}</arg>
```

Comment:
Test is pretty lenient. I guess if we find 1 similar relation we could consider it as failure having 250 input records. 

Although this should be enough for smoke test.